contracts/governance/AxelarMultisigMintLimiter.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IMultisigExecute } from '../interfaces/IMultisigExecute.sol';
+import { MultisigBase } from '../auth/MultisigBase.sol';
+
+contract AxelarMultisigMintLimiter is MultisigBase, IMultisigExecute {
+    constructor(address[] memory accounts, uint256 threshold) {
+        _rotateSigners(accounts, threshold);
+    }
+
+    function execute(address target, bytes calldata callData) external payable onlySigners {
+        (bool success, ) = target.call{ value: msg.value }(callData);
```

Comment:
msg.value can be influenced by the last signer vs having it as an arg. We can assume the contract was pre-funded with that value to let it go through